backend/.gitignore

```diff
@@ -31,3 +31,9 @@ build/
 
```

Comment:
You need to .gitignore .mvn! Do amend your commit so that you remove you pusing those files.